main.js

```diff
@@ -0,0 +1,420 @@
+topicQuestion.innerHTML = questions[currentQuestionNumber].topic;
+
+// Show Questions
+function showQuestions() {
+  let element = document.getElementById('question');
+  element.innerHTML = questions[currentQuestionNumber].question;
+  if ('imgURL' in questions[currentQuestionNumber]) {
+    let y = document.createElement('br');
+    element.appendChild(y);
+
+    let imgObj = document.createElement('img'); // <img> <img1>
+    imgObj.setAttribute('src', questions[currentQuestionNumber].imgURL);
+    imgObj.setAttribute('alt', 'imgObj');
+    let x = document.getElementById('question');
+    x.appendChild(imgObj);
+  }
+}
+
+const nextButton = document.getElementById('next-btn');
+const backButton = document.getElementById('back-btn');
+
+// onclick of button-next go to the next question and save the answer
+nextButton.addEventListener('click', function() { showNextQuestion(); saveTheAnswer() });
+
+backButton.addEventListener('click', showPreviousQuestion);
+
+function showPreviousQuestion() {
+  currentQuestionNumber = currentQuestionNumber - 1;
+  pointsCounter--;
+  showQuestions();
+  showChoices();
+  hideNextButton();
+  if (currentQuestionNumber > 0) {
+    showBackButton();
+  } else {
+    hideBackButton();
+  }
+  hideSubmitButton();
+
+  showProgress();
+}
+
+function showNextQuestion() {
+  currentQuestionNumber = currentQuestionNumber + 1;
+  showProgress();
+  showQuestions();
+  showChoices();
+  hideNextButton();
+  if (currentQuestionNumber > 0) {
+    showBackButton();
+  } else {
+    hideBackButton();
+  };
+
+  // let choiceButton = document.getElementById('btn' + i);
+  // choiceButton.onclick = function() {
+  // if (currentQuestionNumber === questions.length - 1) {
+  //   showSubmitButton();
+  // } else {
+  //   hideSubmitButton();
+  // }
+
+}
+
+
+// Submit Button
+const submitButton = document.getElementById('submit-btn');
+
+submitButton.addEventListener('click', showResults);
+
+// let resultPage = document.createElement('results');
+
+
+function showResults() {
+  let resultPage = document.getElementById('results');
+  document.getElementById('quiz').style.display = 'none';
+
+  let headerResults = document.createElement('h1');
+  let text = document.createTextNode('Your results:');
+  headerResults.appendChild(text);
+  resultPage.appendChild(headerResults);
+
+  let rightAnswers = document.createElement('p');
+  let textAns = document.createTextNode(
+    `${pointsCounter} out of ${questions.length}`
+  );
+  rightAnswers.appendChild(textAns);
+  resultPage.appendChild(rightAnswers);
+
+  let enteredPoints = `${pointsCounter}`;
+
+  let totalPoints = questions.length;
+  let percent = (enteredPoints * 100) / totalPoints;
+
+  let userPercent = document.createElement('p');
+  let userPercentElement = document.createTextNode(percent + ' %');
+  userPercent.appendChild(userPercentElement);
+  resultPage.appendChild(userPercent);
+
+
+  if (percent >= 80) {
+    let p1 = document.createElement('p');
```

Comment:
see how the functionality repeats for each case (percent > 80, > 45, else) ? maybe consider creating a function that creates the textNode and the Image and appends them accoringly, and use it for each case :)